src/components/CustomSlider.jsx

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable react/jsx-filename-extension */
+/* eslint-disable max-len */
+/* eslint-disable react/button-has-type */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/interactive-supports-focus */
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
try to figure out these rules (google them) and avoid `eslint-disable` if it's possible